src/app/layouts/side-nav/side-nav.component.scss

```diff
@@ -0,0 +1,118 @@
+@import "./src/assets/_config"; // import shared colors etc.
```

Comment:
is this file used at all?

Reply:
Yes, it is. It cointains all the variables (color, fonts).

Reply:
haha I don't mean config, I mean `src/app/layouts/side-nav/side-nav.component.scss`

Reply:
AFAIK this component wasn't used at all, but @kewde IMO moved the sidebar nav there to make use of it (+ originally the sidebar was in the `app.component` there it didn't make any sense).

Reply:
Yeah the sidenav css is scattered around, I moved some of it to the component..